src/encoded/audit/experiment.py

```diff
@@ -57,162 +57,177 @@ def audit_experiment_chipseq_control_read_depth(value, system, files_structure):
                                        control_to_target,
                                        target_investigated_as):
     marks = pipelines_with_read_depth['ChIP-seq read mapping']
-    # treat this file as control_bam - raising insufficient control read depth
-    if is_control_file is True:
-        if target_name not in ['Control-human', 'Control-mouse']:
-            detail = ('Control {} file {} ' +
-                     'has a target {} that is neither ' +
-                     'Control-human nor Control-mouse.').format(
-                         value['output_type'],
-                         value['@id'],
-                         target_name)
-            yield AuditFailure('inconsistent target of control experiment', detail, level='WARNING')
-            return
-
-        if control_to_target == 'empty':
-            return
-
-        # control_to_target in broad_peaks_targets:
-        elif 'broad histone mark' in target_investigated_as:
-            if 'assembly' in value:
-                detail = ('Control {} file {} mapped to {} assembly has {} ' +
-                    'usable fragments. ' +
-                    'The minimum ENCODE standard for a control of ChIP-seq assays targeting broad ' +
-                    'histone mark {} ' +
-                    'is 35 million usable fragments, the recommended number of usable ' +
-                    'fragments is > 45 million. (See /data-standards/chip-seq/ )').format(
-                        value['output_type'],
-                        value['@id'],
-                        value['assembly'],
-                        read_depth,
-                        control_to_target
-                    )
-            else:
-                detail = ('Control {} file {} has {} ' +
-                    'usable fragments. ' +
-                    'The minimum ENCODE standard for a control of ChIP-seq assays targeting broad ' +
-                    'histone mark {} ' +
-                    'is 35 million usable fragments, the recommended number of usable ' +
-                    'fragments is > 45 million. (See /data-standards/chip-seq/ )').format(
-                        value['output_type'],
-                        value['@id'],
-                        read_depth,
-                        control_to_target
-                    )
-            if read_depth >= marks['broad']['minimal'] and read_depth < marks['broad']['recommended']:
+    if control_to_target == 'empty':
+        return
+
+    if not control_bam_details:
+        detail = ('The peaks file {} produced by ENCODE uniformly processing '
+                  'ChIP-seq pipeline has no control alignments specified.').format(
+                      peaks_file_id
+                  )
+        yield AuditFailure('missing control alignments', detail, level='ERROR')
+        return
+    control_details = generate_control_bam_details_string(control_bam_details)
+    prefix = ('Control alignment files ({}) '
```

Comment:
add this in an else block after the 'if assembly'

Reply:
require re-demo

Reply:
done